src/PhpDoc/ResolvedPhpDocBlock.php

```diff
@@ -149,7 +151,7 @@ public static function createEmpty(): self
 	{
 		// new property also needs to be added to merge()
 		$self = new self();
-		$self->phpDocString = '/** */';
```

Comment:
because I don't want to change this magic-value for BC reasons, I added a new `hasPhpDocString` to determine whether a phpdoc is user-defined or not